picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
Again, I think this would be overkill. If you don't save a file because nothing has changed you don't rename files, delete directories etc.

BUT, if you previously saved images to files and now save them to tags (or vice versa) then the file **has** changed. We would be better concentrating on loading file metadata more comprehensively, and then checking whether it has changed - so for images we should load files and tags (noting their source at the same time) and then see if the images have changed c.f. downloaded.